qdrant/v0.11.0.x/distributed_deployment.md

```diff
@@ -194,4 +194,111 @@ After that, Qdrant will exclude the node from the consensus, and the instance wi
 
 ## Replication
 
-Currently work-in-progress, see [Roadmap](https://qdrant.to/roadmap)
+*Since version v0.11.0*, Qdrant allows to replicate shards between nodes in the cluster.
+
+Shard replication increases the reliability of the cluster by keeping several copies of a shard spread among the cluster.
+This ensure the availability of the shards in case of node failures except of all replicas are lost.
```

Comment:
This ensures the availability of the shards in case of node failures, except if all replicas are lost.